app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -110,6 +110,49 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
     return result.data;
   }
 
+  Future<List<dynamic>> _queryCirrusGraphQL(
+    String sha,
+    GraphQLClient client,
+  ) async {
+    assert(client != null);
+    const String owner = 'flutter';
+    const String name = 'flutter';
+    const String cirusStatusQuery = r'''
+    query BuildBySHAQuery($owner: String!, $name: String!, $SHA: String) { 
+      searchBuilds(repositoryOwner: $owner, repositoryName: $name, SHA: $SHA) { 
+        id latestGroupTasks { 
+          id name status 
+        } 
+      } 
```

Comment:
Should this now be moved to a common query file to share between both places?

Reply:
Created a query file: refresh_cirrus_status_queries